espnet2/asr/espnet_model.py

```diff
@@ -46,7 +49,7 @@ def __init__(
         encoder: AbsEncoder,
         decoder: AbsDecoder,
         ctc: CTC,
-        rnnt_decoder: None,
+        rnnt_decoder: Optional[AbsDecoder],
```

Comment:
Should we use the name of `rnnt` in rnnt_decoder? transducer_decoder is more general?

Reply:
ditto